modules/ROOT/pages/mongodb/mongodb-connector-6-0-examples.adoc

```diff
@@ -0,0 +1,106 @@
+= MongoDB Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+== Invoke an Operation
+
+To invoke a simple operation, such as the *Insert Document* operation, follow these steps:
```

Comment:
Just checking: I have seen other "operation"s named in this topic set that were not uniformly init. capped and not boldface. What plan do you follow? We are trying to document one clear method in the style ref.